src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java

```diff
@@ -0,0 +1,53 @@
+package stsjorbsmod.cards.cull.deckoftrials;
+
+import com.megacrit.cardcrawl.actions.common.ExhaustSpecificCardAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.tempCards.Smite;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Blasphemer extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Blasphemer.class);
+
+    private static final CardRarity RARITY = CardRarity.SPECIAL;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = -2;
+    private static final int SMITE_AMOUNT = 5;
+    private static final int UPGRADE_SMITE_AMOUNT = 1;
+
+    public Blasphemer() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        this.exhaust = true;
+        this.cardsToPreview = new Smite();
+        magicNumber = baseMagicNumber = SMITE_AMOUNT;
+    }
+
+    @Override
+    public void triggerWhenDrawn() {
+        this.addToTop(new MakeTempCardInHandAction(new Smite(), this.magicNumber));
+        this.addToTop(new ExhaustSpecificCardAction(this, AbstractDungeon.player.hand));
+    }
+
+    @Override
+    public void upgrade() {
+        if (!this.upgraded) {
+            this.upgradeName();
+            this.upgradeMagicNumber(UPGRADE_SMITE_AMOUNT);
```

Comment:
We like to include `this.upgradeDescription()` in all cards' `upgrade` methods; it doesn't really cost anything noticeable to do it when it's not needed, and it's a good practice because there are some rare cases where English doesn't need a special upgrade description but a different language might want one.